src/routes/transactions/entities/transaction.entity.ts

```diff
@@ -34,7 +35,10 @@ export class Transaction {
   })
   txInfo: TransactionInfo;
   @ApiPropertyOptional({
-    oneOf: [{ $ref: getSchemaPath(MultisigExecutionInfo) }],
+    oneOf: [
+      { $ref: getSchemaPath(MultisigExecutionInfo) },
+      { $ref: getSchemaPath(ModuleExecutionInfo) },
```

Comment:
I think you need to include the entity definition `ModuleExecutionInfo` in the `@ApiExtraModels` decorator as well. Otherwise, the info won't be shown in Swagger UI:

<img width="273" alt="Screenshot 2022-12-29 at 17 29 25" src="https://user-images.githubusercontent.com/21182069/209982059-d0772643-baa3-4d13-8c46-05a77792b337.png">

Reply:
Done https://github.com/5afe/safe-client-gateway-nest/pull/205/commits/cb10fc45deab30d28bf84f288052aae7287963c4 but for me swagger is not loading the schema.

Reply:
Maybe it's a browser problem? I see it properly now after the change:
<img width="616" alt="Screenshot 2022-12-30 at 13 22 43" src="https://user-images.githubusercontent.com/21182069/210069791-ccc6c47e-f812-4251-ad70-7d032c76873f.png">

Reply:
thanks!